control-plane/agents/src/bin/core/controller/scheduling/volume_policy/pool.rs

```diff
@@ -13,17 +19,36 @@ impl PoolBaseFilters {
     pub(crate) fn capacity(request: &GetSuitablePoolsContext, item: &PoolItem) -> bool {
         item.pool.capacity > request.size
     }
+    /// Helper util for overcommit checks.
+    fn overcommit_util(allowed_commit_percent: u64, pool: &PoolWrapper, required: u64) -> bool {
```

Comment:
Why not leave this as overcommit and add the calling helper to the caller, example, to `GetSuitablePoolsContext` and to `ReplicaResizePoolsContext`.

```suggestion
    fn overcommit(allowed_commit_percent: u64, pool: &PoolWrapper, required: u64) -> bool {
```

Reply:
added as method in respective structs.